cypress/integration/Auth/Register.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { last } from "cypress/types/lodash";
```

Comment:
If single quote, single quote for all

Reply:
**Prettier** should have done this.
I will look at how to apply the react npm script to do this.

Reply:
https://prettier.io/docs/en/precommit.html